.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
```

Comment:
Should we include a link to the 1Password Developers Slack workspace in this list?